### ISSUE TITLE
Problem with Token Layer 

### ISSUE THREAD

**Issue by theczarek** [NeV3rKilL/alien-actions#1]
Alien RPG system: Moving (dragnig) token after condition activation causes token to freze and inactive. 

Console erroro message:

> Uncaught (in promise) Error: Invalid BaseTexture modules/alien-actions/assets/fast-action.svg
>     loadTexture foundry.js:35374
>     _drawEffect foundry.js:44999
>     drawEffects foundry.js:44956
>     _refreshTokens foundry.js:12422
>     _onCreateEmbeddedDocuments foundry.js:12393
>     _handleCreateEmbeddedDocuments foundry.js:8654
>     _createEmbeddedDocuments foundry.js:8544
>     create backend.mjs:95
>     createDocuments document.mjs:332
>     create document.mjs:431
>     _toggleActiveEffect foundry.js:45378
>     toggleEffect foundry.js:45333
>     _onToggleEffect foundry.js:30893
>     jQuery 8
>     activateListeners foundry.js:30749
>     _render foundry.js:2070
>     call_wrapped libWrapper-wrapper.js:465
>     <anonymous> listeners.js:75
>     _render#0 libWrapper-wrapper.js:160
>     _render foundry.js:28262
>     render foundry.js:2004
>     _refreshTokens foundry.js:12425
>     _onDeleteEmbeddedDocuments foundry.js:12409
>     _handleDeleteEmbeddedDocuments foundry.js:9036
>     _deleteEmbeddedDocuments foundry.js:8942
>     delete backend.mjs:214
>     deleteDocuments document.mjs:402
>     delete document.mjs:463
>     _toggleActiveEffect foundry.js:45365
>     toggleEffect foundry.js:45333
>     _onToggleEffect foundry.js:30893
> foundry.js:35374:28

![obraz](https://user-images.githubusercontent.com/83540404/128419233-8b39d5be-1597-4147-b8a2-f11984eb54c7.png)

**Comment by Gorvaine** [comment]
I could not reproduce this issue.

Seems that your file "modules/alien-actions/assets/fast-action.svg" is corrupted. 

I doublechecked and the current file i github is ok.

Try deleting and reinstalling the module to ensure that the file is ok.

Thanks

**Comment by theczarek** [comment]
I'v tried reinstalling module, also checked manualy if file is in designated forler. I've done that before submiting issue. 
I'v done some test on new game i've created. 
Problem occurs every time i do drag and drop move on character tooken that has at least one of module's action status. I've tested it inside and outside combat -the same effect. No difrence if i use macro or add status manualy.
Also status icons do not appear on tooken like in showcase.
Foundry version 8.8, working from GM account.
Test game has only Reverse Initiative Order and Fast Actions modules installed.

**Comment by Gorvaine** [comment]
Are you using TheForge or other similar service? They could be messing with the assets.

If so, i'll contact them to find the best way to include the assets.

Oh, and "Reverse Initiative" is not needed anymore. The Alien System create different turns for creatures with movement > 1, and use ascending order on initiative by default.

**Comment by theczarek** [comment]
I'm using SSD cluod service unicloud.pl.

I'v done some more testing and it seem problem is related to hosting method. On my local machine mod works fine.

Problem appears only on cloud hosting. 
Other status icons works fine everywhere. Including core foundry icons as well as motion tracker indicator from MT mod. 
My guess is that '-' in file name is restricted on this version of linux system. But it's longshot.

Also with update to v1.4.2 of Fast Actions mod macros are gone form compedium. It's empty - on both cloud and local host.

**Comment by Gorvaine** [comment]
I fixed the compendium and removed special characters on the images and created a new working version 1.4.3

Can you check if this fixes the issue with your cloud service?

https://github.com/NeV3rKilL/alien-actions/commit/8abb493c3f0ac350275d7554f2f930d79e0447d0#diff-c6d09e06c2ca86001d5469aa82a0ff003e787f7ef00504ffabb1f55176403452

https://github.com/NeV3rKilL/alien-actions/commit/8141cb6ec26988ee738614ce75fc3774cf7fe4db#diff-8bf729ffe074caee622c02928173467e658e19e28233cff8a445819e3cae4d50

**Comment by theczarek** [comment]
Nope. That didn't helpled (Long shot as i said).

I fixed problem other way. Svg files was corputed in some way. Windows version accepted them but linux not (or mabe i was a browser)  I took conditions svg's from foudry and renamed them for module svg's. They worked fine. So i edited them in Inkscape removing oryginal shapes and copy pasting yours. Now they are rendered on tooken proprely and don't freeze tookens after move.

Also mind that simply opening your orginal files and saving with Inkscape didnt work. I had to take foundry file. 

Here are the files that work in my case.
[Alien.zip](https://github.com/NeV3rKilL/alien-actions/files/6950622/Alien.zip)
